pkg/plugins/plugins.go

```diff
@@ -41,11 +41,20 @@ type plugins struct {
 	plugins map[string]*Plugin
 }
 
+type extpointHandlers struct {
+	sync.RWMutex
+	extpointHandlers map[string][]func(string, *Client)
+}
+
 var (
-	storage          = plugins{plugins: make(map[string]*Plugin)}
-	extpointHandlers = make(map[string][]func(string, *Client))
+	storage  = plugins{plugins: make(map[string]*Plugin)}
+	handlers = extpointHandlers{}
 )
 
+func init() {
+	handlers.extpointHandlers = make(map[string][]func(string, *Client))
```

Comment:
nit, any reason this isn't done when the variable is declared? (Just occurred to me)